logistics/scripts/modules/database/database-lifecycle.psm1

```diff
@@ -136,7 +136,9 @@ function Initialize-EdFiDatabaseWithDbDeploy {
 
         [string] $createByRestoringBackup,
 
-        [string] $msSqlBackupPath
+        [string] $msSqlBackupPath,
+
+        [Int] $databaseTimeoutInSeconds = 60
```

Comment:
there should probably only be one function with a default so that we don't have to update it in multiple places

Reply:
Both Invoke-DbDeploy and Initialize-EdFiDatabaseWithDbDeploy are used in other places where we are not reading from appsettings. That's why I have added default for both.

Reply:
updated